### PR TITLE
ref(ui) Use ButtonBar in more places

### PR DESCRIPTION
**Pull request by markstory**
Replace the `btn-group` LESS class in more places while also replacing `btn` with Button component, and LESS icons with SVG icons at the same time.

### CODE REVIEW

**Comment by priscilawebdev** [comment]
@markstory some Percy screenshots don't look great, for example:

image


Could you please check that out?

File: src/sentry/static/sentry/app/components/pagination.tsx [hunk 1]
```diff
@@ -48,39 +51,30 @@ class Pagination extends React.Component<Props> {
...
-    if (links.next.results === false) {
-      nextPageClassName += ' disabled';
-    }
-
     return (
       <div className={className}>
-        <div className="btn-group">
-          <a
+        <ButtonBar merged>
+          <Button
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/pagination.tsx]: 👏It's better to have a button here instead of a link ... thanks for this

**Comment by markstory** [comment]
The missing pagination chevrons are a bit of a mystery. The render fine in local testing and when I run acceptance tests locally. I'm going to see if a rebuild clears them up.

**Comment by markstory** [comment]
@priscilawebdev Percy diffs look better. Could you take another look.

File: src/sentry/static/sentry/app/views/eventsV2/eventDetails/pagination.tsx [hunk 2]
```diff
@@ -62,74 +62,40 @@ const Pagination = (props: Props) => {
...
+    <Paginator merged>
+      <Button
+        size="small"
         to={links.oldest || ''}
         disabled={links.previous === null || links.oldest === null}
-      >
-        <InlineSvg src="icon-prev" />
-      </StyledIconLink>
-      <StyledTextLink
+        icon={<IconPrevious size="xs" />}
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/eventsV2/eventDetails/pagination.tsx]: @markstory so the icons are only broken in `app/components/pagination`? 👀

Comment (markstory) [reply src/sentry/static/sentry/app/views/eventsV2/eventDetails/pagination.tsx]: No the chevron icons don't render in percy anywhere. The same icon is used in the sidebar for the collapse sidebar item. That sidebar item has no icons in any percy diff.

Comment (billyvg) [reply src/sentry/static/sentry/app/views/eventsV2/eventDetails/pagination.tsx]: Ah the chevron -- I just checked Percy and there seems to be a firefox upgrade, I'll see if the sidebar icon shows up in future builds or not.

@doralchan any ideas why this icon wouldn't be rendering? I don't know what version of firefox we currently use on percy (but mark checked his local firefox and it renders fine).

**Review by billyvg — APPROVED** [review]
lgtm

**Pull request merged**